ThirdPartyAdapters/liftoffmonetize/liftoffmonetize/src/main/java/com/google/ads/mediation/vungle/rtb/VungleRtbBannerAd.java

```diff
@@ -50,7 +49,6 @@ public class VungleRtbBannerAd implements MediationBannerAd, BannerAdListener {
       mediationAdLoadCallback;
   private MediationBannerAdCallback mediationBannerAdCallback;
 
-  private VungleBannerView bannerAdView;
```

Comment:
QQ. If this is not here, will the reference be released by the garbage collection?